### PR TITLE
Fiddle UI improvements

### PR DESCRIPTION
**Pull request by steve8708**
Screen Shot 2021-06-01 at 7 07 25 AM

### CODE REVIEW

**Comment by steve8708** [comment]
@Dkendal for some reason I can't get tests to pass - unable to require.resolve the json modules, have you seen this before or have any ideas?

**Comment by steve8708** [comment]
CLI tests don't seen to work on `main` branch either so maybe we can ignore in regards to this PR as it's unrelated and worth a look separately

**Comment by Dkendal** [comment]
@steve8708 the require.resolve issues are resolved, the other errors look (more) legit, did something change with the jsx-lite/core build process?

**Comment by steve8708** [comment]
Shouldn't have no. I'm seeing different errors locally (trying to run .d.ts files as tests and complaining you cannot have files with no tests)

**Comment by Dkendal** [comment]
I ran into that as well it was just a left over from running TSC and then getting compiled for some reason. You can delete any .d.ts and js files there. If there are any js files they'll break the tests as they ll get sourced instead of the ts files.

**Comment by Dkendal** [comment]
Alternatively run`git clean packages/cli/src -i`

**Comment by steve8708** [comment]
Still am having issues getting it to work - can you take over on getting cli tests passing when time permits?

**Comment by Dkendal** [comment]
Sure thing

**Comment by Dkendal** [comment]
@steve8708 fixed

File: packages/fiddle/src/index.tsx [hunk 1]
```diff
@@ -2,22 +2,26 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App';
-import reportWebVitals from './reportWebVitals';
 import { configure } from 'mobx';
 
 configure({
   enforceActions: 'never',
+  // useProxies: 'ifavailable',
```
Comment (Dkendal) [inline packages/fiddle/src/index.tsx:9]: :scissors:

**Review by Dkendal — APPROVED** [review]

File: packages/fiddle/src/index.tsx (hunk 1 above)
Comment (Dkendal) [inline packages/fiddle/src/index.tsx:9]:
```suggestion
```

**Review by Dkendal — APPROVED** [review]

File: packages/fiddle/src/index.tsx (hunk 1 above)
Comment (steve8708) [reply packages/fiddle/src/index.tsx:9]: oops! nice catch :)

**Comment by steve8708** [comment]
@Dkendal 🙏 🙏🙏

**Pull request merged**